app/controllers/concerns/requires_recent_mfa.rb

```diff
@@ -5,6 +5,7 @@ class MissingMfaMethod < StandardError; end
 
   def has_done_mfa_recently?
     return true if Rails.env.test? && Rails.application.config.allow_insecure_change_credential
+    return true unless current_user.needs_mfa?
```

Comment:
If I followed the booleans correctly, here we're saying that the user has done mfa recently if there's no phone number in the db. Isn't that a bit misleading?

Reply:
It is, I'll rename the concern